pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -370,23 +369,22 @@ cdef class _Timestamp(datetime):
 
     @property
     def _repr_base(self) -> str:
-        return '{date} {time}'.format(date=self._date_repr,
-                                      time=self._time_repr)
+        return f'{self._date_repr} {self._time_repr}'
```

Comment:
can you make this use double-quotes.  we're consistent about this in the non-cython code, would be nice to do here too